sailfishusermanager.cpp

```diff
@@ -695,6 +695,31 @@ uint SailfishUserManager::currentUser()
     return uid;
 }
 
+QString SailfishUserManager::currentUserUUID()
+{
+    const uint uid = currentUser();
+    const QString uuid = m_lu->getUserUUID(uid);
```

Comment:
I guess you can just call `return userUUID(currentUser())` here for code reusage.

Reply:
Yes, we can.